python/src/aac/plugins/help_dump/help_dump_impl.py

```diff
@@ -30,4 +30,5 @@ def get_command_info():
 
 def _get_all_commands() -> list[AacCommand]:
     all_plugins = plugin_manager.get_plugin_manager().get_plugins()
-    return list(flatten([plugin.get_commands() for plugin in all_plugins if hasattr(plugin, "get_commands")]))
+    list_plugins = list(flatten([plugin.get_commands() for plugin in all_plugins if hasattr(plugin, "get_commands")]))
+    return sorted(list_plugins)
```

Comment:
Glad that it sorts by the name attribute by default, but I'd like to make it explicit for future readers. 

```suggestion
    return sorted(list_plugins, key=lambda command: command.name)
```